### DISCUSSION TITLE
Microsoft.Maui.Graphics.Controls Namespace

### DISCUSSION THREAD

**Discussion by robloo** [dotnet/Microsoft.Maui.Graphics.Controls#95] [General]
I realize Microsoft.Maui.Controls is already taken; however, 'Graphics' in the entire history of .NET means primitive shapes and drawing/rendering. Graphics are a lower level than Controls. So the namespace doesn't quite sit right with me. Controls should be a new namespace IMO at least at the same level of Graphics, not as a child.

Microsoft.Maui.DrawnControls would be better.

Honestly MAUI itself should be changed and Microsoft.Maui.Controls should be this repository. Add a Microsoft.Maui.NativeControls or Microsoft.Maui.LegacyControls for the old XF technique. This repository is the future and the namespaces should be future-proofed now while Maui is unreleased.

**Comment by jsuarezruiz** [comment]
Thanks for the feedback!. I agree that there are namespaces that better declare the functionality or that are simpler to use. Being in experimental phase, temporarily the namespace is not really important for now because can change. If the project passes the experimental phase, the namespaces should change and we will take your feedback into account.
